### PR TITLE
pkg_config setting for arm

### PR DESCRIPTION
**Pull request by ami-GS**
## Description

Some install deps using prepare-machines.ps1, some are not.

## Testing

see automation

## Documentation

_Is there any documentation impact for this change?_

### CODE REVIEW

**Comment by nibanks** [comment]
This PR didn't fix the problem. I don't think it's about XDP dependencies. The issue is more about architecture stuff with the openssl build.

**Comment by ami-GS** [comment]
Maybe this is using newer docker image, but using older prepare-machine.ps1/build.ps1 which causes pkg_config side effect

**Comment by ami-GS** [comment]
Ready to go👌

File: scripts/build.ps1 [hunk 1]
```diff
@@ -396,6 +396,12 @@ function CMake-Generate {
                 "arm64" { $Arguments += " -DCMAKE_CXX_COMPILER_TARGET=aarch64-linux-gnu -DCMAKE_C_COMPILER_TARGET=aarch64-linux-gnu -DCMAKE_TARGET_ARCHITECTURE=arm64" }
                 "arm" { $Arguments += " -DCMAKE_CXX_COMPILER_TARGET=arm-linux-gnueabihf  -DCMAKE_C_COMPILER_TARGET=arm-linux-gnueabihf -DCMAKE_TARGET_ARCHITECTURE=arm" }
             }
+
+            # to build with pkg-config
+            switch ($Arch) {
```
Comment (nibanks) [inline scripts/build.ps1:401]: Shouldn't this just be part of the switch right above this?

**Comment by nibanks** [comment]
Can you please update the title/description to match the change now? Thanks!

File: scripts/build.ps1 (hunk 1 above)
Comment (ami-GS) [reply scripts/build.ps1:401]: It becomes diverging the change between main branch. is it okey?

**Pull request merged**